.github/workflows/pytest.yml

```diff
@@ -41,24 +41,14 @@ jobs:
           pip install -r requirements_dev.txt
       - name: Delete default runner images
         run: |
-          docker image rm node:14
```

Comment:
Why should we delete those lines? those images are no longer pulled by barge?

Reply:
they don't exist anymore in github runner

Reply:
see https://github.com/oceanprotocol/provider/actions/runs/7810951839/job/21305187944